generators/server/templates/build.gradle.ejs

```diff
@@ -32,6 +32,7 @@ plugins {
     id "com.github.node-gradle.node"
 <%_ } _%>
     id "org.sonarqube"
+    id "com.diffplug.spotless" version "<%- javaDependencies['spotless-gradle-plugin'] %>"
```

Comment:
Just to be consistent, can you move the version number to `pluginManagement` block in `settings.gradle`, such that we can use it here without the version?

Reply:
yes, sure :)

Reply:
changed it